Alignment/CommonAlignmentProducer/plugins/PCLTrackerAlProducer.cc

```diff
@@ -507,7 +507,7 @@ ::createGeometries(const edm::EventSetup& setup)
     setup.get<PTrackerParametersRcd>().get( ptp );
 
     theTrackerGeometry = boost::shared_ptr<TrackerGeometry>(
-        trackerBuilder.build(&(*geometricDet), *ptp )
+        trackerBuilder.build(&(*geometricDet), *ptp, tTopo )
```

Comment:
@ianna - I can see that the Tracker Topology is built from part of them, but unfortunately I cannot find an easy way to access to the ones that are used into the TrackerGeomBuilder. A possibility can be introduce them as private members in the class TrackerTopologyEP  and then building the Tracker Topology with them.. Any suggestion / comment?